clj-refactor.el

```diff
@@ -1415,7 +1415,7 @@ optionally including those that are declared private."
 (defun cljr--name-of-current-def ()
   (cljr--goto-toplevel)
   (ignore-errors (forward-char))
-  (when (looking-at "def")
+  (when (looking-at "\\(.+/\\)?def")
```

Comment:
this is not an easy diff to read. generally speaking better to have indentation fix commits separately in the future. not a biggie tho :)